_posts/2016-03-16-Containers-all-the-way-through.md

```diff
@@ -24,7 +24,7 @@ tags:
 - operating system
 --- 
 
-### Introduction
+<br/>
```

Comment:
Yes otherwise the tags and intro text look squashed together, maybe something in the template could help